bundles/core/pom.xml

```diff
@@ -375,6 +375,11 @@
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.i18n</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This will be needed all the time? Or will this dependency be moved into the AEM uber jar later?￼

Reply:
In the long term it will be in the uber jar/sdk, yes.

Reply:
+1 lets add a TODO comment here to remove later